pwndbg/commands/heap.py

```diff
@@ -619,6 +620,10 @@ def vis_heap_chunks(addr=None, count=None, naive=None):
         # it's difficult (impossible?) to find all the thread caches for a
         # specific heap.
         bin_collections.insert(0, allocator.tcachebins(None))
+    bin_collections.insert(0, allocator.fastbins(None))
```

Comment:
Can we just put this in the `bins_collection` array above?

Reply:
Why add these at all?
They cause free chunk labels to be printed twice, as shown in your screenshot.